manager/integration/tests/common.py

```diff
@@ -4984,6 +4984,54 @@ def get_volume_running_replica_cnt(client, volume_name):  # NOQA
     return cnt
 
 
+def create_rwx_volume_with_storageslass(client,
+                                        core_api,
+                                        storage_class):
+
+    VOLUME_SIZE = str(DEFAULT_VOLUME_SIZE * Gi)
+
+    pvc_name = generate_volume_name()
+
+    pvc_spec = {
+        "apiVersion": "v1",
+        "kind": "PersistentVolumeClaim",
+        "metadata": {
+                "name": pvc_name,
+        },
+        "spec": {
+            "accessModes": [
+                "ReadWriteMany"
+            ],
+            "storageClassName": storage_class['metadata']['name'],
+            "resources": {
+                "requests": {
+                    "storage": VOLUME_SIZE
+                }
+            }
+        }
+    }
+
+    core_api.create_namespaced_persistent_volume_claim(
+        'default',
+        pvc_spec
+    )
+
+    check_pvc_in_specific_status(core_api, pvc_name, 'Bound')
+
+    for i in range(RETRY_EXEC_COUNTS):
+        volume_name = get_volume_name(core_api, pvc_name)
+        if volume_name is not None:
+            break
+        time.sleep(RETRY_INTERVAL)
```

Comment:
What is the purpose of this for loop?

Reply:
Hi @c3y1huang, this part is to prevent in case volume name not ready, but I use master-head to run 20 times of this script. found that this loop is not really necessary, I removed, thank you